src/convert.ts

```diff
@@ -22,26 +23,100 @@ import {
   tsNullKeyword,
   tsNumberKeyword,
   tsPropertySignature,
+  tsQualifiedName,
+  TSQualifiedName,
   tsStringKeyword,
   tsThisType,
   tsTupleType,
   TSType,
+  tsTypeAliasDeclaration,
   tsTypeAnnotation,
+  TSTypeElement,
   tsTypeLiteral,
   tsTypeParameter,
   tsTypeParameterDeclaration,
+  tsTypeParameterInstantiation,
   tsTypeQuery,
   tsTypeReference,
   tsUndefinedKeyword,
   tsUnionType,
-  tsVoidKeyword,
+  tsUnknownKeyword,
   TypeAnnotation,
-  TypeParameter
+  TypeParameter,
+  ObjectTypeIndexer,
+  tsIndexSignature,
+  TSTypeParameterDeclaration,
+  TSTypeParameter,
+  TSTypeAliasDeclaration,
+  TypeAlias,
+  TSTypeAnnotation,
+  InterfaceDeclaration,
+  TSInterfaceDeclaration,
+  tsInterfaceDeclaration,
+  InterfaceExtends,
+  TSExpressionWithTypeArguments,
+  tsInterfaceBody,
+  ObjectTypeAnnotation,
+  ObjectTypeProperty,
+  TSPropertySignature
 } from '@babel/types'
 import { generateFreeIdentifier } from './utils'
 
-// TODO: Add overloads
+export function typeAliasToTsTypeAliasDeclaration(
+  node: TypeAlias
+): TSTypeAliasDeclaration {
+  if (node.type !== 'TypeAlias') {
+    throw new Error('Expected TypeAlias!')
+  }
+  const typeParameters: TSTypeParameterDeclaration | null = node.typeParameters
```

Comment:
No need for explicit annotation.

```suggestion
  const typeParameters = node.typeParameters
```